glib/src/gstring.rs

```diff
@@ -892,63 +892,32 @@ impl FromGlibPtrBorrow<*mut i8> for GString {
     }
 }
 
-#[doc(hidden)]
-impl<'a> ToGlibPtr<'a, *const u8> for GString {
```

Comment:
Why? Also you then probably want to do the same for the impls for `String`, `&str` etc

Reply:
I seem to remember we replaced this with `c_char` somewhere else? I can add the `u8` and `i8` back, not sure it makes a difference

Reply:
I think it's mostly for convenience. `c_char` is sometimes `u8`, sometimes `i8` (depending on platform), and simply having both trait impls available here reduces the potential of mismatches.

Reply:
I just took that commit out